eng/common/scripts/Verify-Readme.ps1

```diff
@@ -0,0 +1,15 @@
+# Wrapper Script for Readme Verification
+[CmdletBinding()]
+param (
+  [string]$DocWardenVersion = "0.7.1",
```

Comment:
Cool I like that this is defaulted and I much prefer it here instead of needing to set it in all the language repos.